tests/contract_testcases/polkadot/inheritance/inherit_types_05.sol

```diff
@@ -0,0 +1,22 @@
+contract b {
+    struct foo {
+        uint32 f1;
+        uint32 f2;
+    }
+}
+
+contract c {
+    enum foo {
+        f1,
+        f2
+    }
+}
+
+contract a is b, c {
+    function test(foo x) public {}
+}
+
+// ---- Expect: diagnostics ----
+// error: 1:1-6:2: contracts without public storage or functions are not allowed on Polkadot. Consider declaring this contract abstract: 'abstract contract b'
```

Comment:
Upper case P on polkadot, while it is writing with a lower case in other diagnostics.